test/cmocka/src/debugability/macros.c

```diff
@@ -46,7 +51,7 @@ static void test_debugability_macros_declare_log_entry(void **state)
 			"1"
 			"0"
 			"1"
-			"31"
+			"36"
```

Comment:
Is this related ? if so, what does it do ?

Reply:
I have a strong suspicion this fixes https://sof-ci.01.org/sof-pr-viewer/#/build/PR5126/build8185406

Reply:
ok, but @aiChaoSONG @marc-hb do we need a conditional around this change as I assume it was working before ?

Reply:
I agree there is at least a commit message gap right now.

Reply:
@lgirdwood Yes, it is related. the number 31/36 here is the line number, the correct line number to make the test pass is always the line number of `const char *should_be_eq =` MINUS 1. if I add/remove lines above  `const char *should_be_eq =`,  the line number should be changed accordingly, or the test will fail.

@marc-hb  https://sof-ci.01.org/sof-pr-viewer/#/build/PR5126/build8185406 this link is the old one, the link for the latest code is https://sof-ci.01.org/sof-pr-viewer/#/build/PR5126/build8186409, and the test passed.